packages/protocol-parser/src/common.ts

```diff
@@ -51,3 +52,5 @@ export type ValueArgs = {
   encodedLengths: Hex;
   dynamicData: Hex;
 };
+
+export type PartialTable = Pick<Table, "schema" | "key">;
```

Comment:
should probably give this a better name if we're going to use it outside the context of a single module